src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3RequestTests.cs

```diff
@@ -537,6 +537,58 @@ public async Task POST_ServerAbort_ClientReceivesAbort(HttpProtocols protocol)
         }
     }
 
+    [ConditionalFact]
+    [MsQuicSupported]
+    public async Task POST_ServerAbortAfterWrite_ClientReceivesAbort()
+    {
+        // Arrange
+        var builder = CreateHostBuilder(async context =>
+        {
+            Logger.LogInformation("Server writing content.");
+            await context.Response.Body.WriteAsync(new byte[16]);
+
+            // Note that there is a race here on what is sent before the abort is processed.
+            // Abort may happen before or after response headers have been sent.
+            Logger.LogInformation("Server aborting.");
```

Comment:
Do you want to wait until the response is received before aborting? Or are intentionally covering both cases here?

Reply:
It's intentional. There are other tests that wait for the response.